src/CLU/Microsoft.Azure.Commands.Websites/Content/help/az.hlp

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
@arroyc as we discussed yesterday Replace this please. with something like welcome to the azure management commands.  Run 'azure help' to get a list of all commands."